sky/shell/gpu/mojo/rasterizer_mojo.h

```diff
@@ -16,14 +16,18 @@
 namespace sky {
 namespace shell {
 
-class RasterizerMojo : public Rasterizer {
+class RasterizerMojo : public ::sky::shell::Rasterizer,
+    private rasterizer::Rasterizer {
```

Comment:
Yeah, if we move the inheritance up, then we won't have to repeat ourselves here.